updatePomVersions.sh

```diff
@@ -0,0 +1,33 @@
+# this script updates all maven projects versions (maven-java and samples) to align with the current sdk version
+# if SDK_VERSION env var is defined, it will use it, otherwise it will take the version from sbt
+# after running this script, you may run local tests or simply send a PR with the updates. 
+# the script can also run publishM2 and mvn install to generate the artifacts with the new version.
```

Comment:
With this script we can test locally, but also update the pom after a release. We don't need to do it on each release though.